bindings/wasm/Cargo.toml

```diff
@@ -33,6 +33,7 @@ default-features = false
 features = ["account", "bindings-derive"]
 
 [dev-dependencies]
+json = { version = "0.12.4" }
 wasm-bindgen-test = { version = "0.3" }
```

Comment:
Do we absolutely need this `json` dev-dependency? Is it for convenience or is there something that `serde_json` cannot do?